src/util/address.rs

```diff
@@ -210,8 +213,11 @@ impl Address {
     pub fn script_pubkey(&self) -> script::Script {
         match self.payload {
             Payload::Pubkey(ref pk) => {
+                let mut hash = Vec::new();
```

Comment:
`hash` seems to be a misleading name here.

Reply:
Good catch, fixed!